src/social-network/worker.ts

```diff
@@ -12,6 +12,18 @@ import getCurrentNetworkWorker from './utils/getCurrentNetworkWorker'
 export interface SocialNetworkWorkerDefinition extends SocialNetworkWorkerAndUIDefinition {
     /**
      * This function should fetch the given post by `fetch`, `AutomatedTabTask` or anything
+     * @pseudoCode
+     * fetchPostContent(post) {
+     *      let tab = get_tab_with_same_origin_and_not_pinned()
```

Comment:
actually it doesn't have to be unpinned

Reply:
why? this is required when I tried to implement this...
or I cannot understand what it does.